contracts/service-provider-directory/src/contract/execute.rs

```diff
@@ -128,15 +180,12 @@ pub(crate) fn delete_nym_address(
     let services_to_delete = query::query_nym_address(deps.as_ref(), nym_address)?.services;
```

Comment:
Just making sure I understood it correctly, the number of  services that could be returned by this is bound by `MAX_NUMBER_OF_ALIASES_FOR_NYM_ADDRESS`, right?

Reply:
Correct